api/src/main/java/org/openmrs/api/impl/VisitServiceImpl.java

```diff
@@ -352,18 +352,16 @@ public VisitAttribute getVisitAttributeByUuid(String uuid) {
 	@Override
 	public void stopVisits(Date maximumStartDate) {
 		
-		List<VisitType> visitTypesToStop = getVisitTypesToStop();
+		getVisitTypesToStop();
```

Comment:
@Ankitkumar94 keep them in the local variable instead of querying the method a few times, also make the variable final.